src/structures/UserProfile.js

```diff
@@ -41,6 +41,12 @@ class UserProfile {
      * @type {boolean}
      */
     this.premium = data.premium;
+    
+    /**
+     * If the user is a Discord Partner
```

Comment:
Perhaps "Whether the user is a Discord partner" would be better, this more in-line with the rest of the docs.